google/cloud/iam/CMakeLists.txt

```
@@ -53,7 +53,8 @@ target_include_directories(
 target_link_libraries(
     google_cloud_cpp_iam
     PUBLIC google-cloud-cpp::grpc_utils google-cloud-cpp::common
-           google-cloud-cpp::iam_protos google-cloud-cpp::iam_v1_policy_protos)
+           google-cloud-cpp::iam_protos google-cloud-cpp::iam_v1_policy_protos
+           google_cloud_cpp_iam_policy_protos)
```

For consistency, maybe we should use the aliased name?

```suggestion
           google-cloud-cpp::iam_policy_protos)
```